### PR TITLE
Add capability to generate CH4 benchmark output

### PR DESCRIPTION
**Pull request by msulprizio**
This PR adds the benchmark type "CH4Benchmark" for validating GEOS-Chem CH4 simulations. This option utilizes the existing benchmark code and a reduced set of options from the FullChemBenchmark type (concentration plots, emission plots, emission tables, mass tables, OH metrics).

### CODE REVIEW

File: gcpy/benchmark.py [hunk 1]
```diff
@@ -1747,14 +1756,16 @@ def make_benchmark_emis_tables(
     # ==================================================================
 
     # Emissions species dictionary
-    species = yaml.load(
+    spc_dict = yaml.load(
```
Comment (yantosca) [inline gcpy/benchmark.py:1759]: Maybe better here to use `spc_dict = util.read_config_file(os.path.join(os.path.dirname(__file__), emission_spc.yml))`

**Review by yantosca — CHANGES_REQUESTED** [review]
I've flagged a place where we can use the `util.read_config_file` function to read the config file instead of using `yaml.load` etc, to be consistent with other usage.

Other than this, looks great to merge!

File: gcpy/benchmark.py (hunk 1 above)
Comment (msulprizio) [reply gcpy/benchmark.py:1759]: I'm still seeing several uses of `yaml.load` in this file:

```
158:     properties = yaml.load(open(properties_path), Loader=yaml.FullLoader)
427:     properties = yaml.load(open(properties_path), Loader=yaml.FullLoader)
1777:    spc_dict = yaml.load(
1782:    inv_dict = yaml.load(
2404:    newvars = yaml.load(
3385:    spcdb = yaml.load(open(path), Loader=yaml.FullLoader)
3395:    aod = yaml.load(open(path), Loader=yaml.FullLoader)
3826:    spc_properties = yaml.load(open(os.path.join(os.path.dirname(__file__),
4208:    properties = yaml.load(open(properties_path), Loader=yaml.FullLoader)
```

Would you recommend changing all of these? If so, maybe that's best done in a separate PR?

Comment (yantosca) [reply gcpy/benchmark.py:1759]: Yeah maybe let's do a separate PR.

Comment (msulprizio) [reply gcpy/benchmark.py:1759]: OK, thanks. I just submitted a feature request so we don't forget. See https://github.com/geoschem/gcpy/issues/188.

**Review by yantosca — COMMENTED** [review]
Changes have been applied in GCPy 1.3.3.

**Pull request merged**